mapit/shortcuts.py

```diff
@@ -33,47 +40,64 @@ def default(self, o):
 def render(request, template_name, context=None):
     if context is None:
         context = {}
-#    context['base'] = base or 'base.html'
-#    context['connection'] = connection
     return render_to_response(
         template_name, context, context_instance=RequestContext(request)
     )
 
 
-def sorted_areas(areas):
-    # In here to prevent a circular dependency
-    from mapit import countries
-    if hasattr(countries, 'sorted_areas'):
-        return countries.sorted_areas(areas)
-    return list(areas)
-
-
 def output_html(request, title, areas, **kwargs):
     kwargs['json_url'] = request.get_full_path().replace('.html', '')
     kwargs['title'] = title
-    kwargs['areas'] = sorted_areas(areas)
     kwargs['indent_areas'] = kwargs.get('indent_areas', False)
```

Comment:
Yeah, but then I need to pass it in the `template_area` iterator so thought I might as well leave it for that.